.github/workflows/pr-automation.yml

```
@@ -0,0 +1,91 @@
+  SLEEP_SECONDS: '30' # Interval for checking CodeRabbit status
+
+jobs:
+  wait-for-coderabbit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Wait for CodeRabbit review completion
+        id: wait_coderabbit
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:14:8: Double quote to prevent globbing and word splitting

(shellcheck)